src/models/order.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model, SchemaTypes } = require("mongoose");
+
+const schema = new Schema({
+    productionDay: {
+        type: Date,
+        required: true,
+        unique: true,
+        length: 7
+    },
+    deliveryDay: {
+        type: Date,
+        required: true,
+        unique: true,
+        length: 7
+    },
+    restaurantId: {
```

Comment:
do not use `restaurantId` but `restuarant` because when you populate this thanks to Mongoose's schema, it'll fill in the restaurantId property with a restaurant object instance....